src/Controls/samples/Controls.Sample.UITests/Elements/ScrollViewCoreGalleryPage.cs

```diff
@@ -31,18 +31,18 @@ public ScrollViewCoreGalleryPage(ScrollOrientation orientation = ScrollOrientati
 			var btnStack2 = new StackLayout { Orientation = StackOrientation.Horizontal };
 			var btnStack3 = new StackLayout { Orientation = StackOrientation.Horizontal };
 
-			var btn = new Button { Text = "Scroll to 100" };
-			var btn4 = new Button { Text = "Scroll to 100 no anim" };
-			var btn1 = new Button { Text = "Start" };
-			var btn2 = new Button { Text = "Center" };
-			var btn3 = new Button { Text = "End" };
+			var btn = new Button { AutomationId = "Scroll100", Text = "Scroll to 100" };
```

Comment:
Is this because we can't find elements based on Text?

Reply:
Yes, the Click method currently only finds elements marked with AutomationId. We have several gaps that we are covering little by little. For example:
- Set device orientation: https://github.com/dotnet/maui/pull/19100
- Scroll with gesture: https://github.com/dotnet/maui/pull/19181
- Etc

Since there is a workaround, a simple change in the test, it has been applied but we will implement the ability to click by searching for the element by content.